src/pages/AboutUs/ManagementCommittee/index.tsx

```diff
@@ -0,0 +1,13 @@
+import styles from './ManagementComittee.module.css'
+import { AspectRatio, Button, ButtonGroup } from '@chakra-ui/react'
+import { Image } from '@chakra-ui/react'
```

Comment:
Can you please change the import ordering to follow this convention (and also in all other React files):

1. React and external libraries (including chakra ui, react-router, etc)
2. utility files (/api, /constants, /hooks, etc)
3. Custom components (/components)
4. styles

<img width="763" alt="Screen Shot 2022-02-07 at 22 09 56" src="https://user-images.githubusercontent.com/69668484/152804643-32841c90-e981-4b6a-b9be-6719c3fba8d7.png">

Reply:
Done!